tendermint/src/lite.rs

```diff
@@ -3,3 +3,122 @@ pub mod verifier;
 
 pub use self::types::*;
 pub use self::verifier::*;
+
+#[cfg(test)]
+mod tests {
+    use super::super::rpc::endpoint::commit::SignedHeader;
+    use super::super::{block, validator};
+    use super::*;
+    use serde_json;
+
+    #[test]
+    fn check_verifier_with_mock_data() {
+        let header: block::Header = serde_json::from_str(
+            r#"{
+        "version": {
+          "block": "10",
+          "app": "0"
+        },
+        "chain_id": "test-chain-y3m1e6-AB",
+        "height": "1",
+        "time": "2019-11-19T07:56:45.46738Z",
+        "num_txs": "0",
+        "total_txs": "0",
+        "last_block_id": {
+          "hash": "",
+          "parts": {
+            "total": "0",
+            "hash": ""
+          }
```

Comment:
This will panic: 
https://github.com/interchainio/tendermint-rs/blob/ef7410c184c736f032f24296c24cf921175582be/tendermint/src/amino_types/block_id.rs#L36-L37
called e.g. by:
https://github.com/interchainio/tendermint-rs/blob/ef7410c184c736f032f24296c24cf921175582be/tendermint/src/block/header.rs#L106

Reply:
This is a bug fixed in another PR, https://github.com/interchainio/tendermint-rs/pull/66.